Tests/Common/Data/SliceTests.cs

```diff
@@ -206,6 +207,122 @@ public void AccessesGenericallyByTypeAndSymbol()
             Assert.AreEqual(unlinkedDataSpy, unlinkedData);
         }
 
+        [Test]
+        public void MergeSlice()
+        {
+            TradeBar tradeBar1 = new TradeBar { Symbol = Symbols.SPY, Time = DateTime.Now };
+            TradeBar tradeBar2 = new TradeBar { Symbol = Symbols.AAPL, Time = DateTime.Now, Open = 23 };
```

Comment:
nit - `var`. Should avoid using `DateTime.Now` is possible an use `DateTime.UtcNow` instead, avoid the tz conversion and will already return utc. Ideally get time at the top once an re use it? or even just create a fixed `DateTime` instance